tools/packaging/packaging.psm1

```diff
@@ -1825,8 +1825,7 @@ function New-ILNugetPackage
         "Microsoft.PowerShell.SDK.dll",
         "Microsoft.WSMan.Management.dll",
         "Microsoft.WSMan.Runtime.dll",
-        "System.Management.Automation.dll",
-        "Microsoft.PowerShell.MarkdownRender.dll")
```

Comment:
This file should be signed in your build, so can you remove it from our signing?